modules/aceMedical/preinit.sqf

```diff
@@ -0,0 +1,78 @@
+
+	FNC_assignMedic = {	//Make a unit a medic, also works for medical vehicles. Ex: [this,2] call FNC_assignMedic; makes a unit or medical vehicle a 'doctor' level.
```

Comment:
I recommend adding documentation for these functions to settings.sqf for easier access and less confusion.